examples/text-generation/run_generation.py

```diff
@@ -73,6 +73,11 @@ def setup_parser(parser):
         action="store_true",
         help="Whether to use the key/value cache for decoding. It should speed up generation.",
     )
+    parser.add_argument(
+        "--use_torch_compile",
```

Comment:
```suggestion
        "--torch_compile",
```
to be aligned with Transformers and GaudiTrainingArguments

Reply:
ok. I would change.